### PR TITLE
Various fixes for swift 4

### LINKED ISSUE #14: Update swift4  to allow more than 128KB of input parameters
The epilogue uses environment variable to take the input JSON
```swift
let inputStr: String = env["WHISK_INPUT"] ?? "{}"
```
We should modify this and pass the input as STDIN to allow larger input data (i.e. 1MB, 3MB, etc..) using swift `readLine()`

### LINKED ISSUE #16: Error with urlSession double free or corruption (fasttop) upgrade to swift 4.0.3
Doing a non blocking invoke using urlSession it fails with
```
Starting test Swift Whisk SDK tests using swift:4 should allow Swift actions to invoke other actions and not block at 2018-02-03 21:04:23.588
check failed for activation 9c418ab0f5424923818ab0f5420923ff: 


{
"logs":[
"2018-02-04T02:04:26.134158018Z stdout: Compiling","2018-02-04T02:04:26.134202437Z stdout: swiftc status is 0",
"2018-02-04T02:04:26.13420729Z  stdout: Linking",
"2018-02-04T02:04:26.65737652Z  stdout: ",
"2018-02-04T02:04:26.658329845Z stderr: *** Error in `/swift4Action/spm-build/.build/release/Action': double free or corruption (fasttop): 0x00007f4960000910 ***"]
}


Exception occurred during test execution: java.util.NoSuchElementException: key not found: activationId
```

It looks like is a known issue and fixed in 4.0.1+
https://bugs.swift.org/browse/SR-5936
https://bugs.swift.org/browse/SR-5972
```
Seems fixed in the latest 4.0.1 snapshot.
```
We need to pick up the new image version `ibmcom/swift-ubuntu:4.0.3`

### PR DESCRIPTION
**Pull request by csantanapr**
Fixes #16 with urlSesson double free or corruption
Fixes #14 with large paramters

### CODE REVIEW

**Comment by csantanapr** [comment]
FYI @jthomas

File: tests/src/test/scala/actionContainers/Swift4ActionContainerTests.scala [hunk 1]
```diff
@@ -91,4 +91,21 @@ class Swift4ActionContainerTests extends SwiftActionContainerTests {
...
+      val code = """
+                   | func main(args: [String: Any]) -> [String: Any] {
+                   |     return args
+                   | }
+                   |""".stripMargin
+
+      val (initCode, initRes) = c.init(initPayload(code))
+      initCode should be(200)
+
+      val arg = JsObject("arg" -> JsString(("a" * 198144)))
```
Comment (dubeejw) [inline tests/src/test/scala/actionContainers/Swift4ActionContainerTests.scala]: Hmm, double check that this is actually a MB.

File: tests/dat/build.sh [hunk 2]
```diff
@@ -6,4 +6,6 @@ set -e
 
 ../../tools/build/compile.sh  HelloSwift4 swift:4 "-v"
 
+../../tools/build/compile.sh  SwiftyRequest swift:4 "-v"
+
```
Comment (dubeejw) [inline tests/dat/build.sh]: Too many new lines here.

File: tests/src/test/scala/actionContainers/Swift4ActionContainerTests.scala (hunk 1 above)
Comment (dubeejw) [reply tests/src/test/scala/actionContainers/Swift4ActionContainerTests.scala]: Is this 1024 * 1024?

Comment (csantanapr) [reply tests/src/test/scala/actionContainers/Swift4ActionContainerTests.scala]: humm checking...

Comment (csantanapr) [reply tests/src/test/scala/actionContainers/Swift4ActionContainerTests.scala]: Looks likes the highest I can go without an margin of error is `1048561` will update test

**Comment by csantanapr** [comment]
@dubeejw take a look now

**Pull request merged**